### DISCUSSION TITLE
Unable to get working with SSL in development environment

### DISCUSSION THREAD

**Discussion by kcrwfrd** [i18next/next-i18next#2155] [Q&A]
We're using SSL in our dev environment, Next.js configured to use a custom server similar to what's described here: https://medium.com/@greg.farrow1/nextjs-https-for-a-local-dev-server-98bb441eabd7

Unfortunately after adding next-i18next this no longer works. I've pored through the documentation and source code and haven't been able to find anything regarding getting SSL working... would appreciate any help pointing me in the right direction 🙏🏼

**Comment by adrai** [comment]
I don't think next-i18next does influence SSL... you may ask the https://github.com/vercel/next.js team/repo

**Comment by kcrwfrd** [comment]
Following up on this, yes you're correct. The problem didn't have anything to do with next-i18next, it was actually introduced with Next 13.4 when using custom server setup in your dev environment: https://github.com/vercel/next.js/issues/49355#issuecomment-1537536063

Thank you for responding 🙏🏼
